plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -62,8 +63,15 @@ def get_one_batch_of_trials(
     parallelism: Tuple[int, int],
     num_trials_so_far: int,
     num_max_trials_to_do: int,
-) -> BatchOfTrialType:
-    """Produce a batch of trials that can be run in parallel"""
+) -> Tuple[BatchOfTrialType, bool, str]:
```

Comment:
use a small dataclass for the return value.
returning a complex tuple makes things more cumbersome in my opinion. (especially with your tendency for long variable names :) ).

Reply:
Done!